flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_FWC.js

```diff
@@ -57,16 +57,30 @@ class A32NX_FWC {
         // ESDL 1. 0.320
         this.memoLdgInhibit_conf01 = new NXLogic_ConfirmNode(3, true); // CONF 01
 
-        // master warning & caution buttons
+        // Master Warning & Caution
         this.warningPressed = false;
         this.cautionPressed = false;
+        this.warningPriority = false;
 
-        // altitude warning
+        // Altitude Warning
         this.previousTargetAltitude = NaN;
         this._wasBellowThreshold = false;
         this._wasAboveThreshold = false;
         this._wasInRange = false;
         this._wasReach200ft = false;
+
+        // Autopilot Warning
+        this.AutothrottleWarningCanceled = true;
+        this.AutopilotWarningCanceled = true;
+        this.athrdeltaTime = 0;
+        this.apdeltaTime = 0;
+
+        // AUTOPILOT DISCONNECT
+        this.ATHRDisconnectByThrottle = true;
+        this.APDisconnectedBySidestick = true;
```

Comment:
**issue**: Inconsistent casing.
**suggestion**: `this.camelCasing`, `this.apDisconnectedBySidestick`, `this.apDeltaTime`, etc.